include/byggja/physic.hh

```
@@ -0,0 +1,170 @@
+#ifndef BYGGJA_PHYSIC_HH_
+# define BYGGJA_PHYSIC_HH_
+
+# include <functional>
+# include <initializer_list>
+# include <iostream>
+# include <map>
+
+//# include <experimental/filesystem>
+//namespace fs = std::experimental::filesystem;
```

Not really deadcode, but a reminder to handle STL filesystem.
- [ ] Should use cmake define to make compilation fail if both are not available.